ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+            sh:minCount 1 ;
+            sh:node ositrace:ContentShape ;
+            sh:order 3 ;
+            sh:path ositrace:content ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:QualityShape ;
+            sh:order 4 ;
+            sh:path ositrace:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:QuantityShape ;
+            sh:order 5 ;
+            sh:path ositrace:quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:DataSourceShape ;
+            sh:order 6 ;
+            sh:path ositrace:dataSource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path ositrace:georeference ] ;
+    sh:targetClass ositrace:OSITrace .
+
+ositrace:ContentShape a sh:NodeShape ;
+    sh:property [ skos:example "[motorway, residential]" ;
+            sh:datatype xsd:string ;
+            sh:description "Covered/used road types, defined over ODR element t_road_type, see ODR spec section 8.3"@en ;
+            sh:in ("Bicycle" "LowSpeed" "Motorway" "Pedestrian" "Rural" "Town" "TownArterial" "TownCollector" "TownExpressway" "TownLocal" "TownPlayStreet" "TownPrivate" "Unknown") ;
+            sh:message "Validation of roadTypes failed!"@en ;
+            sh:name "roadTypes"@en ;
+            sh:order 0 ;
+            sh:path ositrace:roadTypes ],
+        [ skos:example "shoulder, curb, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered lane types, see ODR spec section 9.5.3."@en ;
+            sh:in ("biking" "border" "connectingRamp" "curb" "driving" "entry" "exit" "median" "none" "offRamp" "onRamp" "parking" "restricted" "shoulder" "slipLane" "stop" "walking") ;
+            sh:message "Validation of laneTypes failed!"@en ;
+            sh:name "laneTypes"@en ;
+            sh:order 1 ;
+            sh:path ositrace:laneTypes ],
+        [ skos:example "trees, street lamps, road patches..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered object classes, see ODR spec section 11"@en ;
+            sh:in ("barrier" "bike" "building" "bus" "car" "crosswalk" "gantry" "motorbike") ;
+            sh:message "Validation of levelOfDetail failed!"@en ;
+            sh:name "levelOfDetail"@en ;
+            sh:order 2 ;
+            sh:path ositrace:levelOfDetail ], 
+        [ skos:example "right-hand traffic" ;
+            sh:datatype xsd:string ;
+            sh:description "Traffic direction, i.e. right-hand or left-hand traffic"@en ;
+            sh:in ("left-hand" "right-hand") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of trafficDirection failed!"@en ;
+            sh:name "trafficDirection"@en ;
+            sh:order 3 ;
+            sh:path ositrace:trafficDirection ],
+        [ skos:example "object list, detection list" ;
```

Comment:
Does the example should state that both apply? This `,` list is probably confusing with `sh:minCount` --> then ther would be 2 times `ositrace:granularity`

Reply:
See comment below on skos:example: I do not think we have any stilized way of using it. The in constraint will prevent misuse at property level.